src/TextField/TextField.js

```diff
@@ -59,6 +59,7 @@ const getStyles = (props, context, state) => {
       outline: 'none',
       backgroundColor: 'rgba(0,0,0,0)',
       color: props.disabled ? disabledTextColor : textColor,
+      cursor: props.disabled ? 'not-allowed' : 'initial',
```

Comment:
@TildeWill 
maybe, it will be good to create new issue (and pull request) for this since this pull-request was already merged few versions ago?